lxd-agent/tests/test_rss_calculation.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -eu
+
+echo "=== RSS Calculation Validation Test ==="
+
+# Run free command to get baseline
+FREE_USED=$(free | awk '/^Mem:/ {print $3}')
+echo "Used memory from free command: $FREE_USED kB"
+
+# Get meminfo values
+MEM_TOTAL=$(awk '/MemTotal:/ {print $2}' /proc/meminfo)
+MEM_FREE=$(awk '/MemFree:/ {print $2}' /proc/meminfo)
+BUFFERS=$(awk '/Buffers:/ {print $2}' /proc/meminfo)
```

Comment:
Those could benefit from being anchored (`/^.../`) in case some similar lines are ever added to `/proc/meminfo`.